src/components/UI/molecules/ButtonRow/ButtonRow.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import StyledLink from '<atoms>/StyledLink/StyledLink';
+import Image from '<atoms>/Image/Image';
+import { boxShadows } from '<variables>/shadows';
+
+/**
+ * @description - ButtonRow Component
+ *
+ * @prop {string} height - height
+ * @prop {string} width - width
+ * @prop {string} boxShadow - boxShadow
+ * @prop {string} altText - altText
+ * @prop {string} images - images
+ *
+ * @return {component} ButtonRow
+ */
+const ButtonRow = ({
+  width,
+  height,
+  boxShadow,
+  altText,
+  images,
+}) => {
+  return (
+    <FlexContainer alignItems="center" justifyContent="center">
+      {renderImages()}
+    </FlexContainer>
+  );
+
+  function renderImages() {
+    return images.map((image, index) => {
+      return (
+        <StyledLink key={index}>
+          <Image
+            imageUrl={image.url}
+            width={width}
+            height={height}
+            boxShadow={boxShadow}
+            altText={altText}
+            margin="xs"
+          />
+        </StyledLink>
+      );
+    });
+  }
+};
+
+ButtonRow.defaultProps = {
+  width: '58px',
+  height: '58px',
+  boxShadow: 'icon',
+};
+
+ButtonRow.propTypes = {
+  width: PropTypes.string,
+  height: PropTypes.string,
+  boxShadow: PropTypes.oneOf(Object.keys(boxShadows)),
+  altText: PropTypes.string,
+  images: PropTypes.array,
```

Comment:
Change this to  images: PropTypes.arrayOf(
      PropTypes.shape({
        url: PropTypes.string.isRequired,
        width: PropTypes.string,
        height: PropTypes.string,
        boxShadow: PropTypes.oneOf(Object.keys(boxShadows)),
        altText: PropTypes.string,
       margin: PropTypes.string,
      })
    ),